pkgs/development/python-modules/cyclonedds-python/default.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+
+, cyclonedds
+, setuptools
+, rich-click
+
+, pytest
+, pytest-cov
+, pytest-mock
+, pytest-asyncio
+}:
+
+buildPythonPackage rec {
+  pname = "cyclonedds-python";
+  version = "0.10.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "eclipse-cyclonedds";
+    repo = pname;
```

```suggestion
    repo = "cyclonedds-python";
```

We like to keep pname and repo semantically detached.